server/services/database/profile.js

```diff
@@ -22,7 +22,28 @@ const getAllProfiles = () => {
 	return pool.query("SELECT * FROM profile").then((result) => result.rows);
 };
 
+const getProfileById = (profileId) => {
+	return pool
+		.query("SELECT * FROM profile WHERE profile_public_id=$1", [profileId])
+		.then((result) => result.rows);
+};
+
+const getEducationById = (profileId) => {
```

Comment:
same as for experience:
- should be in services/database/experience.js
- should be getEducationsByProfileId -> to show that it returns an array based on the profile_id